Packs/Code42/Integrations/Code42/Code42_test.py

```diff
@@ -1792,3 +1800,27 @@ def test_fetch_incidents_fetch_limit(code42_fetch_incidents_mock):
     assert len(incidents) == 1
     assert next_run["last_fetch"]
     assert not remaining_incidents
+
+
+def test_security_data_search_command_searches_exposure_exists_when_no_exposure_type_is_specified(
+        code42_file_events_mock
+):
+    client = create_client(code42_file_events_mock)
+    cmd_res = securitydata_search_command(client, MOCK_SECURITY_DATA_SEARCH_QUERY_WITHOUT_EXPOSURE_TYPE)
+    code42_res = cmd_res[0]
+    file_res = cmd_res[1]
+
+    assert code42_res.outputs_prefix == "Code42.SecurityData"
+    assert code42_res.outputs_key_field == "EventID"
+    assert file_res.outputs_prefix == "File"
+
+    actual_query = code42_file_events_mock.securitydata.search_file_events.call_args[0][0]
+    filter_groups = json.loads(str(actual_query))["groups"]
+
+    # Assert that the  correct query gets made
+    assert len(filter_groups) == 4
```

Comment:
this assertion is somewhat confusing to have in this test (it's hard to understand by looking at this test why it is important there be 4 filter groups). I would copy the same set of assertions that we have in a different test to make this more clear:

```python
   filter_groups = json.loads(str(actual_query))["groups"]
    expected_query_items = [
        ("md5Checksum", "d41d8cd98f00b204e9800998ecf8427e"),
        ("osHostName", "DESKTOP-0001"),
        ("deviceUserName", "user3@example.com"),
        ("exposure", "ApplicationRead"),
    ]
    expected_file_events = json.loads(MOCK_SECURITY_EVENT_RESPONSE)["fileEvents"]

    # Assert that the  correct query gets made
    assert len(filter_groups) == len(expected_query_items)
    for i in range(0, len(filter_groups)):
        _filter = filter_groups[i]["filters"][0]
        assert _filter["term"] == expected_query_items[i][0]
        assert _filter["value"] == expected_query_items[i][1]
```